flow/layers/performance_overlay_layer_unittests.cc

```diff
@@ -35,6 +46,10 @@ static std::string GetGoldenFilePath(int refresh_rate, bool is_new) {
 }
 
 static void TestPerformanceOverlayLayerGold(int refresh_rate) {
+#if !defined(OS_LINUX)
```

Comment:
Can you just make the golden check do its think on all platforms? Or does that fail already on Mac?

Reply:
So, Fuchsia crashes on these tests right now :(  That's why I moved the SKIP.  I'll move this back and see what I can do to have Fuchsia not crash

Reply:
@chinmaygarde : yes, it's already failing because this golden test renders texts.

@arbreng : hopefully the crashing test could help us find a problem in Fuchsia, instead of just giving us a false alarm :)

Reply:
Figured out how to fix these tests on Fuchsia; they can read the input goldens now